README.md

```diff
@@ -166,7 +166,7 @@ See the [Get RAPIDS version picker](https://rapids.ai/start.html) for more OS an
 
 ### Conda - Jetson Nano, TK1, TX2, Xavier, Linux OS
 
-While there are many versions of Anaconda for AArch64 platforms, cuSignal has been tested and supports [conda4aarch64](https://github.com/jjhelmus/conda4aarch64/releases). Conda4aarch64 is also described in the [Numba aarch64 installation instructions](http://numba.pydata.org/numba-doc/latest/user/installing.html#installing-on-linux-armv8-aarch64-platforms). Further, it's assumed that your Jetson device is running a current edition of [JetPack](https://developer.nvidia.com/embedded/jetpack) and contains the CUDA Toolkit.
+In cuSignal 0.15 and beyond, we are moving our supported aarch64 Anaconda environment from [conda4aarch64](https://github.com/jjhelmus/conda4aarch64/releases) to [miniforge](https://github.com/conda-forge/miniforge). Further, it's assumed that your Jetson device is running a current edition of [JetPack](https://developer.nvidia.com/embedded/jetpack) and contains the CUDA Toolkit.
```

Comment:
_Running a current edition_ feels a little ambiguous

Reply:
Good point. Let's "support" 4.3 and newer.

Reply:
Like it!